rust/main/agents/relayer/src/msg/pending_message.rs

```diff
@@ -293,10 +294,29 @@ impl PendingOperation for PendingMessage {
 
         let params = MessageMetadataBuildParams::default();
 
-        let metadata_bytes = match message_metadata_builder
+        let build_metadata_start = Instant::now();
```

Comment:
nit: might be nice to put this metric stuff in helper function or something, just so that the `prepare` logic can stay lean. Either a function in PendingMessage or maybe even internal to MessageMetadataBuilder, happy either way